docs/api/server/Module-Map-Schema.md

```diff
@@ -1,7 +1,109 @@
 [👈 Return to Overview](../API.md)
 
-# Module Map
+# Module Map Schema
 
-Documentation Forthcoming
+[**Holocron Modules**](../API.md#modules) may be developed and versioned in isolation as if they were their own frontend applications. The `one-app` Server relies on a `module-map.json` configuration file to orchestrate all these versioned web experiences together to form a single application. A Module Map contains an object of the following information:
+* Holocron Module Name
+* URLs to Holocron Module Bundles
+* [Subresource Integrity](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity) Hash to verify contents of the Holocron Module Bundle
 
-[☝️ Return To Top](#module-map)
\ No newline at end of file
+**Contents**
+* [Shape](#shape)
+* [Example](#example)
+* [Field Annotations](#field-annotations)
+  * [`key` Field](#key-field)
+  * [`[moduleName]` Field](#moduleName-field)
+  * [`browser` Field](#browser-field)
+  * [`legacyBrowser` Field](#legacyBrowser-field)
+  * [`node` Field](#node-field)
+  * [`url` Field](#url-field)
+  * [`integrity` Field](#integrity-field)
+
+## Shape
+
+```js
+({
+  key: String, // required
```

Comment:
With #37 this is optional so should annotate as such here.

Reply:
Fixed in b02bcb1